drivers/include/ethos.h

```diff
@@ -34,14 +34,27 @@ extern "C" {
 #endif
 
 /* if using ethos + stdio, use STDIO_UART values unless overridden */
-#ifdef MODULE_STDIO_ETHOS
+#ifdef MODULE_STDIO_ETHOS || DOXYGEN
```

Comment:
The CI pointed out that this should be:
```suggestion
#if IS_USED(MODULE_STDIO_ETHOS) || defined(DOXYGEN)
```
You will need to `#include "kernel_defines.h"` to use `IS_USED` macro

Reply:
Updates. Please check